lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2456,10 +2456,155 @@ definitions:
           - 'Baicells Nova-233 G2 OD FDD'
           - 'Baicells Nova-243 OD TDD'
           - 'Baicells Neutrino 224 ID FDD'
+          - 'Baicells Nova-436q TDD'
           - 'Baicells ID TDD/FDD'
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      NeighborFreqList:
```

Comment:
snake case
```suggestion
      neighbor_frequency_list:
```

Reply:
The variable should be neighbor_frequency_list, had modify and update.